openqabot/openqa.py

```diff
@@ -84,7 +81,7 @@ def get_job_comments(self, job_id: int):
             )
             ret = list(map(lambda c: {"text": c.get("text", "")}, ret))
         except Exception as e:
-            (method, url, status_code, *other) = e.args
+            (_, _, status_code, *_) = e.args
```

Comment:
not sure about this as code is less readable

Reply:
I agree with the change. It's a good practice to explicitly mark unused return variables so the change is good.